src/zc/buildout/easy_install.py

```diff
@@ -1768,17 +1771,6 @@ def get_all_files(it):
     return [egg_dir]
 
 
-SLASH_BIN_SLASH = os.path.join('', 'bin', '')
-
-
-def _do_not_keep_leftover(filename):
-    return (
-        filename.endswith('.pyc') or
-        filename.endswith('.pyo') or
-        SLASH_BIN_SLASH in filename
-    )
-
-
```

Comment:
Now that you've removed that logic, I wonder if scripts installed via the distutils scripts mechanism (as opposed to sentry-points) will be left over?

If you want an example of a package that uses scripts, try [ansible-cmdb](https://github.com/fboender/ansible-cmdb).

Reply:
`zc.buildout 2.13.3` does not know how to install the `ansible-cmdb` scripts.
Have you already used both combined ?

Reply:
No, it's just the one package I know that still uses scripts instead of entry points.

(I know because I submitted a PR to make it use entry points a long time ago, but it was never merged.)

Curious that old zc.buildout can't handle scripts.  I thought easy_install supported them?

Reply:
Oh, right, zc.buildout took over the role of generating scripts by itself.

Anyway I don't care whether scripts work, I just wondered if they cause complications for this copy-files-except-ones-we-dont-want piece of logic.